packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx

```diff
@@ -3,11 +3,27 @@ import ReactMarkdown from 'react-markdown'
 
 import { MarkdownPreviewStyles } from './MarkdownPreviewStyles'
 
-export const MarkdownPreview = ({ markdown }: { markdown: string }) => {
+export interface MarkdownPreviewProps {
+  markdown: string
+}
+
+export const MarkdownPreview = ({ markdown }: MarkdownPreviewProps) => {
   return (
     <>
       <MarkdownPreviewStyles />
-      <ReactMarkdown className="markdown-preview">{markdown}</ReactMarkdown>
+      <ReactMarkdown
+        className="markdown-preview"
+        components={{
+          code: ({ children, inline }) => {
+            if (inline) {
+              return <code className="inline-code">{children}</code>
+            }
+            return <code className="in-block-code">{children}</code>
+          },
```

Comment:
```suggestion
            if (inline) {
              return <code>{children}</code>
            }
            return <pre><code>{children}</code></pre>
          },
```

The code block was OK - that should stay.

Also, having two warianst I think it should be safe to drop class names as you can differentiate the type of code from nesting:

```css
/* inline code */
.markdown-preview code { ... }

/* code block */
.markdown-preview pre > code { ... }
```

WDYT?